### PR TITLE
[REEF-840] IIMRUClient does not need to have generic type

### PR DESCRIPTION
**Pull request by dkm2110**
This addressed the issue by
* removing generics from the IIMRU Client class definition and shifting it to Submit function
JIRA:
[REEF-840](https://issues.apache.org/jira/browse/REEF-840)

### CODE REVIEW

**Comment by dkm2110** [comment]
@markusweimer @jwang98052 

this is the pull request for changing generics to Submit function

**Comment by jwang98052** [comment]
start looking it now.

File: lang/cs/Org.Apache.REEF.IMRU/InProcess/InProcessIMRUConfiguration.cs [hunk 1]
```diff
@@ -27,10 +27,7 @@ namespace Org.Apache.REEF.IMRU.InProcess
     /// <summary>
     /// Configuration module for the in-process IMRU.
     /// </summary>
-    /// <typeparam name="TMapInput">The type of the side information provided to the Map function</typeparam>
-    /// <typeparam name="TMapOutput">The return type of the Map function</typeparam>
-    /// <typeparam name="TResult">The return type of the computation.</typeparam>
-    public sealed class InProcessIMRUConfiguration<TMapInput, TMapOutput, TResult> : ConfigurationModuleBuilder
+    public sealed class InProcessIMRUConfiguration: ConfigurationModuleBuilder
```
Comment (jwang98052) [inline lang/cs/Org.Apache.REEF.IMRU/InProcess/InProcessIMRUConfiguration.cs]: space before :

**Comment by jwang98052** [comment]
I completed my pass.

**Comment by dkm2110** [comment]
@jwang98052  I have addressed the comments

**Comment by jwang98052** [comment]
I will test and merge

**Pull request closed without merging**